app/assets/javascripts/admin.js

```diff
@@ -18,15 +18,22 @@ $(document).on('turbolinks:load', function() {
 
   $("#collaborators-query").on("input", function() {
     const xhr = new XMLHttpRequest();
+
+    xhr.open(
+      "GET",
+      "/admin/publications/collaborators?collaborators_query=" + $("#collaborators-query").val(),
+      true
+    );
+
+    // required, otherwise Rails fails with a CORS error even if the request is from the same origin
+    xhr.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
+
     xhr.onreadystatechange = function() {
       if (this.readyState == 4 && this.status == 200) {
         eval(xhr.responseText);
       }
     };
-    xhr.open(
-      "GET",
-      "/admin/publications/collaborators?collaborators_query=" + $("#collaborators-query").val(), true
-    );
+
```

Comment:
I have no idea why we're using old-old-old school `XMLHttpRequest` here.  Since this will only be used by people with somewhat modern browsers, we can use `fetch` which is so much simpler.  try this:

```js
fetch(`/admin/publications/collaborators?collaborators_query=${$("#collaborators-query").val()}`).then((resp)  => {
   eval(resp.responseText);
});
```

I'm not 100% sure if the response (resp var) will have a responseText property though.  You might need to put a `console.log(resp)` or a `debugger` in before the eval to play with it.

Reply:
Updated to use fetch()